data/ecosystems/p/polkadot.toml

```diff
@@ -310,6 +312,9 @@ url = "https://github.com/3Dpass/3DP"
 [[repo]]
 url = "https://github.com/4meta5/ssauce"
 
+[[repo]]
+url = "https://github.com/727-Ventures"
```

Comment:
should have its own ecosystem, can you remove this (since it is not a repo) and create a new toml file for itself and place "PolkaEX" under the `sub_ecosystems` at the top of this file?

Reply:
doesn't exist, not adding